docs/07-faq/023-kubernetes-support.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Does Wing support Kebernetes?
+id: kubernetes-support
+keywords: [faq, supported provisioning engines, winglang, Wing programming language, Wing language, Kubernetes, K8S]
+---
+
+Not yet but we will in the future. Our roadmap is [here](https://docs.winglang.io/status#roadmap).
+
+You can add your vote for Kubernetes on [this issue](https://github.com/winglang/wing/issues/2066) to help increase its priority.
```

Comment:
This issue is general. I think this should probably link to an issue about Kubernetes as a target specifically.

Reply:
There is a comment there about K8S with some votes already